### ISSUE TITLE
Touch and Creates in move_java_name not working properly

### ISSUE THREAD

**Issue by DGollings** [7terminals/puppet-java#12]
Didn't seem worthy of a fork + pull, Couldn't figure out why the module kept wanting to reinstall Java constantly, figured it out
:
For whatever reason (Debian Jessie and Puppet 3.4.3) the $name variable isn't set properly as far as touch is concerned. 
Instead of .puppet_java_deployed it would create the empty file .puppet_java_ which of course would break the creates functionality

Fixed it by changing $name to $source which I also find more descriptive
.puppet_java_jdk-7u55-linux-x64.tar.gz_deployed

Basically exec move_java-$name:
      command => "(..) && touch ${deploymentdir}/.puppet_java_${name}_deployed",
      creates => "${deploymentdir}/.puppet_java_${name}_deployed",
becomes
      command => "(..) && touch ${deploymentdir}/.puppet_java_${source}_deployed",
      creates => "${deploymentdir}/.puppet_java_${source}_deployed",

Thanks,
D
